python/dllib/src/bigdl/dllib/net/net_load.py

```diff
@@ -61,6 +64,10 @@ def _get_py_name(self, jclass_name):
             raise Exception("Not supported type: {}".format(jclass_name))
 
     def _load_ppackage_by_jpackage(self, jpackage_name):
```

Comment:
why do we need this?

Reply:
It's used to redirect the jvalue to the proper python class